addon/mixins/spread.js

```diff
@@ -108,9 +108,11 @@ export default Mixin.create({
    * @param {object} spreadableHash - the hash to spread
    */
   _defineSpreadProperties (spreadProperty, spreadableHash) {
+    const supportedSpreadableTypes = A(['instance', 'object'])
+
     assert(
       `${spreadProperty} requires an Ember object or primitive object`,
-      ['instance', 'object'].includes(typeOf(spreadableHash))
+      supportedSpreadableTypes.includes(typeOf(spreadableHash))
```

Comment:
Which browsers? ;) I know there's a line to be drawn, but when Ember, and Ember CLI, proper draws the line around Phantom too, it's reasonable and nice to do the same thing.